configs/11.0/packages/valgrind/sources

```diff
@@ -20,11 +20,11 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=3.11.0
+ATSRC_PACKAGE_VER=3.13.0
 # Don't update the Valgrind revision without updating the VEX revision too.
 # They have to be in sync.
-ATSRC_PACKAGE_REV=15905
-ATSRC_PACKAGE_VEX_REV=3223
+ATSRC_PACKAGE_REV=16446
+ATSRC_PACKAGE_VEX_REV=3398
```

Comment:
> @gftg85 I think it should be nice using the branch instead the trunk but isn't better send it as a
> different task?

IMO it's a simple change and could be done here, but we can create another task and let someone else work on it.

> @gftg85 There's another thing for the branch use suggestion. We use ATSRC_PACKAGE_VER=3.13.0 following the pattern MAJOR.MINOR.PATCH and the branches uses the pattern REPOSITORY_MAJOR_MINOR_BRANCH. We can replace dots for underscore but what about the PATCH I should trim it or we can use ATSRC_PACKAGE_VER=3.13 instead?

3.13.0 is the version of the release, the branch will have all 3.13 releases that's why it doesn't have the last number.
So the name of the branch doesn't interfere in anything.